core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -988,7 +825,7 @@ private boolean isSubjectVersionDeleted(String subject, int version)
     }
   }
 
-  int getMaxIdInKafkaStore() {
+  public int getMaxIdInKafkaStore() {
```

Comment:
I would pass the interface SchemaRegistry to MasterElector instead of the KafkaSchemaRegistry or use some kind of bridge interface between MasterElector and the SchemaRegistry